avaframe/ana4Stats/getStats.py

```diff
@@ -82,35 +54,53 @@ def extractMaxValues(inputDir, cfgMain, avaDir, varPar, nameScenario='', paramet
     # load dataFrame of all simulation configurations
     simDF = cfgUtils.createConfigurationInfo(avaDir, standardCfg='')
 
-    # load peakFiles of all simulations and generate dictionary
+    # load peakFiles of all simulations and generate dataframe
     peakFilesDF = fU.makeSimDF(inputDir, avaDir=avaDir)
     nSims = len(peakFilesDF['simName'])
+    # initialize peakValues dictionary
     peakValues = {}
     for sName in simDF['simName'].tolist():
         peakValues[sName] = {}
 
-    # Loop through peakFiles and compute probability
+    # Loop through result field files and compute statistical measures
     for m in range(nSims):
-
+        # filter simulations according to parametersDict
         if peakFilesDF['simName'][m] in simNameList:
 
             # Load data
             fileName = peakFilesDF['files'][m]
             simName = peakFilesDF['simName'][m]
-            data = np.loadtxt(fileName, skiprows=6)
-
-            # compute max
-            max = np.amax(data)
-
+            dataFull = IOf.readRaster(fileName)
+
+            # if restrictType, set result field values to nan if restrictType result field equals 0
+            if restrictType != '':
+                peakFilesSimName = peakFilesDF[peakFilesDF['simName'] == simName]
+                fileNamePFD = peakFilesSimName[peakFilesSimName['resType'] == restrictType]['files'].values[0]
```

Comment:
what does PFD stand for?

Reply:
peak flow depth, I called it like this as if pfd is zero - there was no ava so this field is useful to restrict the other result fields too

Reply:
but where do I see that you reed the PFD?

Reply:
I will rename it, it is not PFD you can choose what it is in the ini file restrictType=pfd for example.. by default it is pfd

Reply:
okay now I understand